serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -0,0 +1,241 @@
+
+The name of the `configMap` resource consists of the name of the Data Index Service and the suffix `-properties`, as shown in the following example:
+
+.Example `configMap` resource generated during Data Index Service deployment
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: data-index-properties
+data:
+  application.properties : |-
+    property1=value1
+    property2=value2
+----
+
+The `application.properties` data of the `configMap` resource is mounted in a volume to the container of the Data Index Service. Any runtime properties that you add to the `application.properties` section override the default application configuration properties of the Data Index Service.
+
+When the `application.properties` data of the `configMap` is changed, a rolling update modifies the deployment and configuration of the Data Index Service.
+
+---
+
+=== PostgreSQL persistence with the Data Index Service
+
+Data Index Service does not initialize its database schema automatically. To initialize the database schema, you need to either enable automatic schema initialization using Hibernate or enabling flyway migration enabling QUARKUS_FLYWAY_MIGRATE_AT_START.
+
+Automatic schema initialization using Hibernate can be enabled using the `quarkus.hibernate-orm.database.generation` property. For possible values of `quarkus.hibernate-orm.database.generation` property, see https://quarkus.io/guides/hibernate-orm#quarkus-hibernate-orm_quarkus.hibernate-orm.database.generation[Quarkus docs for database generation options]. The value can also be set as an environment variable `QUARKUS_HIBERNATE_ORM_DATABASE_GENERATION`.
+
+.Example `DataIndex` resource with PostgreSQL persistence enabled (requires Kafka KogitoInfra to be available):
+[source,yaml]
+----
+apiVersion: app.kiegroup.org/v1beta1
+kind: KogitoSupportingService
+metadata:
+  name: data-index
+spec:
+  serviceType: DataIndex
+  image: quay.io/kiegroup/kogito-data-index-postgresql:latest
+  infra:
+    - kogito-kafka-infra
+  env:
+    - name: QUARKUS_DATASOURCE_DB-KIND
+      value: postgresql
+    - name: QUARKUS_DATASOURCE_USERNAME
+      value: postgres
+    - name: QUARKUS_DATASOURCE_PASSWORD
+      valueFrom:
+        secretKeyRef:
+          name: kogito-postgresql
+          key: postgresql-password
+    - name: QUARKUS_DATASOURCE_JDBC_URL
+      value: jdbc:postgresql://kogito-postgresql.default.svc.cluster.local:5432/kogito
+    - name: QUARKUS_HIBERNATE_ORM_DATABASE_GENERATION
+      value:
+    - name: QUARKUS_FLYWAY_MIGRATE_AT_START
+      value: "true"
+
+----
+
+[#data-index-dev-service-details]
+== {data_index_ref} as a Quarkus Development Service
+When you use the {product_name} Process Quarkus extension, a temporary {data_index_ref} Service is automatically provisioned while the Quarkus application is running in development mode. When you use one of the following Quarkus extensions, the Dev Service is set up for immediate use:
+
+.Quarkus extension in {product_name}
```

Comment:
No need to mention kogito-quarkus or processes, these docs are only in the context or serverless workflow.

Reply:
@cristianonicolai I replaced that by kogito-quarkus-serverless-workflow in this case, could it makes sense?